concepts/05 UI Components/Diagram/05 Overview/10 Key Features.md

```diff
@@ -0,0 +1,25 @@
+- **Diagram Creation and Editing**    
```

Comment:
Feature list that I want to have here:
1. Diagram Creation and Editing
2. Data Binding
3. Auto layouts (autoSizeEnabled maybe as well)
4. Basic shape types
5. Custom shape types
6. Connectors and connector points options (all that we provide)
7. Interactive features (resize, drag&drop, rotate?, undo&redo, grouping by containers, snapping to the grid ....)
8. Exporting
9. [Adaptability](https://js.devexpress.com/jQuery/Demos/WidgetsGallery/Demo/Diagram/Adaptability/MaterialBlueLight/)
10. [KBN](https://js.devexpress.com/React/Documentation/Guide/UI_Components/Diagram/Keyboard_Support/)
11. Diagram customization